spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/security/OAuthSecurityConfiguration.java

```diff
@@ -42,12 +43,12 @@ protected void configure(HttpSecurity http) throws Exception {
 		http.antMatcher("/**")
 		.authorizeRequests()
 			.antMatchers(
-				"/security/info**", "/login**", "/dashboard/logout-success-oauth.html",
-				"/dashboard/styles/**", "/dashboard/images/**", "/dashboard/fonts/**",
-				"/dashboard/lib/**").permitAll()
+					"/security/info**", "/login**", "/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/logout-success-oauth.html",
```

Comment:
creating a method that does the concatenantion (maybe even accepting varargs) would make this more readable IMO